.travis.yml

```diff
@@ -1,14 +1,22 @@
+language: ruby
+cache: bundler
+sudo: false
 rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.4
-  - jruby-19mode
+  - 2.2.7
```

Comment:
Off the top of my head I can't remember. Depending on the rails version, we'd have to exclude certain combinations. I can add them back to the matrix and see how they test out if you like?